src/Angor/Avalonia/Angor.Model.Implementation/NetworkConfiguration.cs

```diff
@@ -0,0 +1,231 @@
+using Angor.Shared;
+using Angor.Shared.Models;
+using Blockcore.Networks;
+
+namespace Angor.Model.Implementation;
+
+// TODO: This is copied from Angor.Client
+public class NetworkConfiguration : INetworkConfiguration
```

Comment:
hmm I dont like this, but it is temporary code then I guess it is ok for now

Reply:
Yes, that's why I decided to copy it